package.json

```diff
@@ -45,6 +45,7 @@
     "supertest": "^4.0.2",
     "swagger-ui-express": "^4.0.4",
     "unique-slug": "^2.0.1",
+    "unirest": "^0.6.0",
```

Comment:
@Karlmusingo Why this yet we have `supertest`?

Reply:
I used it for requesting the sendgrid API